.github/workflows/update-tor.yml

```diff
@@ -0,0 +1,40 @@
+name: Tor package updates
+on: [pull_request]
```

Comment:
Sorry I missed this earlier. This job should be timer based so it runs daily and preferably not at the same time SDW nightlies run (6am per https://github.com/freedomofpress/securedrop-client/blob/main/.github/workflows/nightlies.yml#L4)...so how about 7am?

Reply:
It was a timer, I just briefly switched it back just now to test my changes :) I'll set it to 7 AM